Makefile

```diff
@@ -1,11 +1,11 @@
-REBAR3=$(shell which rebar3 || echo ./rebar3)
```

Comment:
I think it would make sense to remove the `rebar3` binary from the repo altogether. 

@uwiger Thoughts?

Reply:
For Travis CI it makes more sense to get the latest stable version of `rebar3` as part of the CI step. This way we wouldn't have to maintain a version of `rebar3` in the repository.